### DISCUSSION TITLE
LabPlot2 update although not necessary

### DISCUSSION THREAD

**Discussion by scib50** [Devolutions/UniGetUI#2980] [Q&A]
helo,
i want ask for help with a problem. i've installed labplot2 (version 2.11.1). uningetui cant recognize, that this latest version is already installed. it want's to update from 2.10.1 to 2.11.1.
Is there a workaround?
Thank you very much for this software und the possibility to get help!
greetings, steffen

**Comment by marticliment** [comment]
Hello,

Running the command `winget upgrade` also shows this issue?

**Reply by scib50** [reply]
Hello, Yes it's the same.

**Reply by marticliment** [reply]
then I am afraid you should contact the package maintainers, as this issue is caused by how the software installer reports itself to the system.

**Comment by Lockszmith-GH** [comment]
The workaround is to ignore this package's version (Right Click --> Skip this version) - this way until there is a new version, you won't get notified.

I have this with a couple of packages where winget doesn't really understand which version is installed.

**Comment by scib50** [comment]
Thank you for your replies. It's a problem of the makers of labplot2. the workaround is a good possability.
